subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -224,6 +225,14 @@ struct pdu_data_llctrl_enc_rsp {
 	u8_t ivs[4];
 } __packed;
 
+struct pdu_data_llctrl_start_enc_req {
+	u8_t  empty[0];
```

Comment:
Does the compiler really complain if you have empty structs (with no members inside them)? I'd have assumed it doesn't.

Reply:
It did not, and am glad, as this allowed me to use offsetof and sizeof values in lookup table.

Reply:
Sorry... you mean you do not want "empty[0];"  ?

Reply:
yes, that's what I meant. If it's not strictly required then it seems a bit pointless.

Reply:
if you want you can replace it with /* no parameters */ or something like that

Reply:
will do.